openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdsender/src/tools/CommandSender/CommandSender.vue

```diff
@@ -497,16 +501,29 @@ export default {
                 if (parameter.format_string) {
                   val = sprintf(parameter.format_string, parameter.default)
                 }
+                let range = 'N/A'
+                // check using != because compare with null
+                if (parameter.minimum != null && parameter.maximum != null) {
+                  if (parameter.data_type === 'FLOAT') {
+                    // This is basically to handle the FLOAT MIN and MAX so they
+                    // don't print out the huge exponential
+                    if (parameter.minimum < -1e6) {
+                      parameter.minimum = parameter.minimum.toExponential(3)
+                    }
+                    if (parameter.maximum > 1e6) {
+                      parameter.maximum = parameter.maximum.toExponential(3)
+                    }
+                  }
+                  range = `${parameter.minimum}..${parameter.maximum}`
+                }
                 this.rows.push({
                   parameter_name: parameter.name,
                   val_and_states: {
                     val: val,
                     states: parameter.states,
-                    selected_state: null,
-                    selected_state_label: '',
-                    manual_value: null,
```

Comment:
Were these all unused?

Reply:
Sort of. They're unused here but used in the parameter editor. I think Vue just populates what you give it so there was no need to pass them here.